### PR TITLE
Fixing almost all e2e cases. Added Mainnet deployment configuration.

### PR DESCRIPTION
**Pull request by Marcin-Radecki**
This PR brings back unpausing MOST on both sides during dev, which was erroneously removed in the other PR. Also, there was some problem with particular method ETH most is unpaused, hence I used the same method which MOST L2 is unpaused.

This fixes all e2e tests except Eth -> L1 AZERO one.

### CODE REVIEW

File: e2e-tests/src/balance.rs [hunk 1]
```diff
@@ -91,13 +91,14 @@ impl Balance {
     /// Bridge AZERO to wAZERO.
     pub fn bridge_azero_azero_to_eth(&self, transfer_amount: u128) -> Result<Self> {
         let mut balance = self.clone();
+        let azero_wzaero_decimal_ratio = 1000000u128;
```
Comment (kostekIV) [inline e2e-tests/src/balance.rs:94]:
```suggestion
        let azero_wzaero_decimal_ratio = 1_000_000u128;
```

Comment (kostekIV) [reply e2e-tests/src/balance.rs:94]: is u128 needed?

File: eth/scripts/1_setup_bridge_contracts.js [hunk 2]
```diff
@@ -80,7 +73,7 @@ async function main() {
   // --- setup
   const Most = await ethers.getContractFactory("Most");
   const most = Most.attach(addresses.most);
-
+  //
```
Comment (kostekIV) [inline eth/scripts/1_setup_bridge_contracts.js:76]: nit: rm

File: eth/scripts/1_setup_bridge_contracts.js [hunk 3]
```diff
@@ -92,17 +85,6 @@ async function main() {
     );
   }
 
-  // --- add Aleph -> Ethereum token pairs
```
Comment (kostekIV) [inline eth/scripts/1_setup_bridge_contracts.js:95]: why this change?

**Review by kostekIV — APPROVED** [review]

File: eth/scripts/1_setup_bridge_contracts.js (hunk 3 above)
Comment (Marcin-Radecki) [reply eth/scripts/1_setup_bridge_contracts.js:95]: We removed this pair when we merged change https://github.com/Cardinal-Cryptography/most/commit/f2bedb59fd98f884ddd4909c29484122508ba81e, so that one must brigde native AZERO with `sendRequestAzeroToNative`

File: eth/scripts/1_setup_bridge_contracts.js (hunk 2 above)
Comment (Marcin-Radecki) [reply eth/scripts/1_setup_bridge_contracts.js:76]: done
